src/tezos_rpc/http.mli

```diff
@@ -0,0 +1,11 @@
+val http_get :
```

Comment:
I feel like only the `http_post` is used, then should we already define it?

Reply:
It's gonna be used in a future PR(which is now already opened), also it's a 2 lines implementation, so it's quite reasonable.